images/capi/packer/ova/ubuntu-2404-efi.json

```
@@ -10,9 +10,9 @@
   "firmware": "efi",
   "floppy_dirs": "",
   "guest_os_type": "ubuntu-64",
-  "iso_checksum": "e240e4b801f7bb68c20d1356b60968ad0c33a41d00d828e74ceb3364a0317be9",
+  "iso_checksum": "d6dab0c3a657988501b4bd76f1297c053df710e06e0c3aece60dead24f270b4d",
   "iso_checksum_type": "sha256",
-  "iso_url": "https://releases.ubuntu.com/noble/ubuntu-24.04.1-live-server-amd64.iso",
+  "iso_url": "https://releases.ubuntu.com/noble/ubuntu-24.04.2-live-server-amd64.iso",
```

Could you please update the URLs to use:
```suggestion
  "iso_url": "https://releases.ubuntu.com/24.04.2/ubuntu-24.04.2-live-server-amd64.iso",
```

Once a new release is put out by ubuntu these will then redirect to the `old-releases` URL and not break image-builder :)

---

See https://github.com/kubernetes-sigs/image-builder/issues/1334 for more info about this

---

Oh thanks, didn't know that. I'll make that change

---

No worries :) It's been annoying me for a while now as those immutable URLs are only available after their initial release. 🤷  

Ping me when you're ready for another review.

---

Looks like Canonical's redirect for `releases.ubuntu.com/24.04.x/...` is broken is redirects to `old-releases.ubuntu.com/22.04.x/...`. Also the latest version of the release doesn't exist on `old-releases.ubuntu.com` so I can either pin the releases to the previous version 24.04.1 and it will stay good, or I can set it to the latest and it will break when .3 comes out. Thoughts?

---

🤦  This is such a pain. Lets just stick with the original change for now. It's no worse of than it already is.

---

C'est la software!